### DISCUSSION TITLE
Lists being ex- and importable from/to a human-readable format (json, toml, yaml)

### DISCUSSION THREAD

**Discussion by simonsan** [community/community#8932] [Other Feature Feedback, Questions, & Ideas]
I would really appreciate this feature, as I would be able to download a list of my starred repositories (3k+), write some small tool to sort it into some lists depending on keywords and import it back again on Github.

**Comment by brianlovin** [comment]
Thanks for the note @simonsan, and I agree that we don't have the right tools in place to help power-star users like yourself backfill and organizing a lot of existing starred repositories. I'll add your +1 to our issues to make progress here!

**Comment by cliffordfajardo** [comment]
@brianlovin - is twitter lists data exposed via the API ?
I also am on the github power user train (3k+ stars), maxed out my twitter lists already

Example of my use case on twitter:

Before GitHub lists, I created my own lists using an external store (notion database) but the need to context switch out of the GitHub web app  to save the repo and categorize it  usually caused me to just defer the action 😢 due to the extra step / time needed specially since I use GitHub on mobile a lot (I use it to catch up on updates / view GitHub feed like I do my twitter feed )

**Reply by brianlovin** [reply]
Looks like Lists data is available through their API https://developer.twitter.com/en/docs/twitter-api/v1/accounts-and-users/create-manage-lists/api-reference/get-lists-list

Agreed it would be awesome to have better tools on our end 😊

**Comment by simonsan** [comment]
Small addition maybe @brianlovin: Would be extra nice if the exported data would contain the topics of the repositories, so we could parse that and create lists based on that. Maybe that's even another feature suggestion in general to implement on your side: 

**Give the opportunity to sort into or create lists from your Github stars based on the topics of the repositories.**

**Comment by ahillio** [comment]
Even just being able to export the data (without the import tool) would be nice. Is there no way to download our own list data?
